openreview/tools.py

```diff
@@ -626,17 +615,12 @@ def recruit_reviewer(client, email, first,
 
     hash_seed: a random number for seeding the hash.
     recruit_message: a formattable string containing the following string variables:
-        name
-        accept_url
-        decline_url
+    name
```

Comment:
I think these tabs should stay, or there should be some other way of differentiating `name`, `accept_url`, and `decline_url` from the function variables (these three are variables to the formattable string, not the function)